core/embed/rust/src/ui/model_mercury/component/homescreen/mod.rs

```diff
@@ -208,63 +192,7 @@ impl Component for Homescreen {
     }
 
     fn paint(&mut self) {
-        self.pad.paint();
```

Comment:
I think we can remove `pad` entirely since it should not be needed with new drawing library? Perhaps also `paint_notification_only`?

Reply:
removed both in 8079534d0e1e039b83f407e8ed62f14d0b078430